src/bin/cql-stress-cassandra-stress/settings/command/read_write.rs

```diff
@@ -0,0 +1,395 @@
+    match unit {
+        'k' => 1_000,
+        'm' => 1_000_000,
+        'b' => 1_000_000_000,
+        _ => panic!("Invalid operation count unit: {unit}"),
+    }
+}
+
+fn parse_operation_count(n: &str) -> u64 {
+    let last = n.chars().last().unwrap();
+    let mut multiplier = 1;
+    let mut number_slice = n;
+    if last.is_alphabetic() {
+        multiplier = parse_operation_count_unit(last);
+        number_slice = &n[0..n.len() - 1];
+    }
+    number_slice.parse::<u64>().unwrap() * multiplier
+}
+
+fn parse_duration_unit(unit: char) -> u64 {
+    match unit {
+        's' => 1,
+        'm' => 60,
+        'h' => 60 * 60,
+        _ => panic!("Invalid duration unit: {unit}"),
+    }
+}
+
+fn parse_duration(n: &str) -> u64 {
+    let multiplier = parse_duration_unit(n.chars().last().unwrap());
+    n[0..n.len() - 1].parse::<u64>().unwrap() * multiplier
+}
+
+fn prepare_parser(cmd: &str) -> (ParamsParser, ReadWriteParamHandles) {
+    let mut parser = ParamsParser::new(cmd);
+    let err = parser.simple_param(
+        "err<",
+        r"^0\.[0-9]+$",
+        Some("0.02"),
+        "Run until the standard error of the mean is below this fraction",
+        false,
+    );
+    let ngt = parser.simple_param(
+        "n>",
+        r"^[0-9]+$",
+        Some("30"),
+        "Run at least this many iterations before accepting uncertainty convergence",
+        false,
+    );
+    let nlt = parser.simple_param(
+        "n<",
+        r"^[0-9]+$",
+        Some("200"),
+        "Run at most this many iterations before accepting uncertainty convergence",
+        false,
+    );
+    let no_warmup =
+        parser.simple_param("no-warmup", r"^$", None, "Do not warmup the process", false);
+    let truncate = parser.simple_param(
+        "truncate=",
+        r"^(never|once|always)$",
+        Some("never"),
+        "Truncate the table: never, before performing any work, or before each iteration",
+        false,
+    );
+    let cl = parser.simple_param(
+        "cl=",
+        "^(one|quorum|local_quorum|each_quorum|all|any|two|three|local_one|serial|local_serial)$",
+        Some("local_one"),
+        "Consistency level to use",
+        false,
+    );
+    let serial_cl = parser.simple_param(
+        "serial-cl=",
+        "^(serial|local_serial)$",
+        Some("serial"),
+        "Serial consistency level to use",
+        false,
+    );
+    let n = parser.simple_param(
+        "n=",
+        r"^[0-9]+[bmk]?$",
+        None,
+        "Number of operations to perform",
+        true,
+    );
+    let duration = parser.simple_param(
+        "duration=",
+        r"^[0-9]+[smh]$",
+        None,
+        "Time to run in (in seconds, minutes or hours)",
+        true,
+    );
+
+    parser.group(vec![
+        &err, &ngt, &nlt, &no_warmup, &truncate, &cl, &serial_cl,
+    ]);
+    parser.group(vec![&n, &no_warmup, &truncate, &cl, &serial_cl]);
+    parser.group(vec![&duration, &no_warmup, &truncate, &cl, &serial_cl]);
```

Comment:
I know it is described in `ParamsGroup`, but a (repeat) comment here would be helpful:
```
/// Usage: read [err<?] [n>?] [n<?] [no-warmup] [truncate=?] [cl=?] [serial-cl=?]
///  OR
/// Usage: read n=? [no-warmup] [truncate=?] [cl=?] [serial-cl=?]
///  OR
/// Usage: read duration=? [no-warmup] [truncate=?] [cl=?] [serial-cl=?]
```